models/refund_db.go

```diff
@@ -3,6 +3,7 @@ package models
 // RefundResourceDB represents the database refund structure
 type RefundResourceDB struct {
 	RefundId          string `bson:"refund_id"`
+	RefundedAt        string `bson:"refunded_at"`
```

Comment:
I think this should be `time.Time` for consistency with other database items
```suggestion
	RefundedAt        time.Time `bson:"refunded_at"`
```

Reply:
Fixed